Sources/Animator/HeroCoreAnimationViewContext.swift

```diff
@@ -151,7 +151,7 @@ internal class HeroCoreAnimationViewContext: HeroAnimatorViewContext {
       anim = banim
     }
 
-    anim.fillMode = kCAFillModeBoth
+    anim.fillMode = convertToCAMediaTimingFillMode(convertFromCAMediaTimingFillMode(CAMediaTimingFillMode.both))
```

Comment:
Does CAMediaTimingFillMode.both work? seems like you're just converting from a CAMediaTimingFillMode to a string and back.